libnacl/base.py

```diff
@@ -64,8 +64,8 @@ def save(self, path, serial='json'):
             import json
             packaged = json.dumps(pre)
 
-        perm_other = stat.S_IWOTH | stat.S_IXOTH | stat.S_IWOTH
```

Comment:
S_IWOTH twice, leaving out S_IROTH.
Also, improves readability if you order them in standard read, write and execute order.  Also makes it easier to see that it's correct upon visual inspection.